pkg/install/templates/memcache-dep.yaml.tmpl

```diff
@@ -20,7 +20,9 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
       - name: memcached
-        image: memcached:1.5.20
+        image: {{ .ImageRepository }}memcached:1.5.20{{ if .ImagePullSecrets }}
```

Comment:
I think it would be better to use whitespace controllers instead of in-lining the `if`.